src/caret_analyze/common/type_check_decorator.py

```diff
@@ -255,6 +255,24 @@ def _custom_wrapper(*args, **kwargs):
     type_check_decorator = decorator
 
 except ImportError:
+    # NOTE: import pydantic may do before than caret_analyze/common/load_config/init_config()
+    # NOTE: so, this logger is initialized independently.
+    handler = StreamHandler()
+    handler.setLevel(WARN)
+
+    fmt = '%(levelname)-8s: %(asctime)s | %(message)s'
+    formatter = Formatter(
+        fmt,
+        datefmt='%Y-%m-%d %H:%M:%S')
+    handler.setFormatter(formatter)
+
+    logger = getLogger(__name__)
+    logger.setLevel(WARN)
+    logger.addHandler(handler)
+
+    logger.warning('pydantic is not installed or is not the latest version. '
```

Comment:
Because I felt that it would be easier to read if there were line breaks, I consider that the following changes should be made.
```
logger.warning('pydantic is not installed or is not the latest version. \n'
                   'Please install or upgrade pydantic as CARET may not work properly. \n\n')
```

Reply:
The message is divided into two lines(, although this is a different method of realization from the taro-yu proposed) in [8bd5c57](https://github.com/tier4/caret_analyze/pull/484/commits/8bd5c5750f4780c7fd93a7b82bb07413131f7623).